doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -512,10 +725,10 @@ The option :kconfig:option:`CONFIG_PSA_USE_CC3XX_HASH_DRIVER` enables the driver
 
 The configuration of the :ref:`nrf_security_drivers_oberon` is automatically generated based on the user-enabled algorithms in  `HASH configurations`_.
 
-Password-authenticated key agreement configurations
+Password-authenticated key exchange configurations
 ***************************************************
```

Comment:
```suggestion
Password-authenticated key exchange configurations
**************************************************
```

Reply:
I asked Pekka about this, because I wasn't sure. He's surprised this didn't throw a compliance error, because the number of delineators/symbols should match the characters above.